ext/datadog_profiling_native_extension/runtime_stacks.c

```diff
@@ -0,0 +1,411 @@
+#include "extconf.h"
```

Comment:
So we both know the reason why this file lives inside the profiling extension, or what it even does but... since most of our discussion so far was over private slack, nobody else will know.

Thus, I think this deserves a big comment explaining:

* What this is
* Why it ended up living here
* How it doesn't interact with profiling

Also "runtime_stacks" is not very obvious -- may I suggest `crashtracking_runtime_stacks.c` or something among those lines that makes it very clear that this is for crashtracking